aws/src/main/java/org/apache/iceberg/aws/glue/IcebergToGlueConverter.java

```diff
@@ -231,11 +255,33 @@ static void setTableInputInformation(
                 .collect(Collectors.toSet()));
       }
 
-      Optional.ofNullable(properties.get(GLUE_DESCRIPTION_KEY))
-          .ifPresent(tableInputBuilder::description);
+      String description = properties.get(GLUE_DESCRIPTION_KEY);
+      if (description != null) {
+        tableInputBuilder.description(description);
+      } else if (existingTable != null) {
+        Optional.ofNullable(existingTable.description()).ifPresent(tableInputBuilder::description);
+      }
+
+      List<Column> columns = toColumns(metadata);
```

Comment:
I'm thinking we can simplify the implementation by creating `existingColumnMap` first and use it inside `addColumnWithDedupe` method. In this way we don't have to replace the new columns.

Reply:
I merged existing column comment processing into creating column set on to `toColumns` and `addColumnWithDedupe` method